template-coq/src/denote.ml

```diff
@@ -683,7 +688,7 @@ let rec run_template_program_rec (k : Evd.evar_map * Constr.t -> unit)  ((evm, p
        (*                                   typ; t'|])) *)
        k (evm, Constr.mkApp (texistT_typed_term, [|typ; t'|]))
```

Comment:
This needs to change if `existT_typed_term` becomes universe polymorphic (which it should).